Documentation/ApiOverview/ContentElements/ContentElementsWizard.rst

```diff
@@ -2,167 +2,197 @@
+    such a content element. The content element might also appear in another tab.
+
+    Use :ref:`User settings configuration <user-settings>` to effectively ban
+    usage for non-admin users.
 
-The above example adds your plugin to the tab "Plugin" in the content element wizard.
-You can add it to one of the other existing tabs or create a new one.
+You can also remove whole groups of content elements from the wizard:
 
-.. tip::
+..  literalinclude:: _AddingYourOwnContentElements/_page_remove_group.tsconfig
+    :caption: EXT:my_sitepackage/Configuration/Sets/MySet/page.tsconfig
 
-   Look into the module :guilabel:`Info > Page TSconfig` for existing
-   configurations of :typoscript:`mod.wizards.newContentElement.wizardItems`.
+.. _content-element-wizard-page-tsconfig-change:
 
+Change title, description, icon and default values in the wizard
+----------------------------------------------------------------
 
-If you add it to any of the other tabs (other than plugins), you must add
-the name to :typoscript:`show` as well:
+You can use the following page tsconfig properties to change the display
+of the element in the wizard:
 
-.. code-block:: typoscript
-   :caption: EXT:example/Configuration/TsConfig/Page/Mod/Wizards/NewContentElement.tsconfig
+*   :confval:`iconIdentifier <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-iconidentifier>`
+*   :confval:`iconOverlay <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-iconoverlay>`
+*   :confval:`title <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-title>`
+*   :confval:`description <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-description>`
+*   :confval:`tt_content_defValues <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-tt-content-defvalues>`
+*   :confval:`saveAndClose <t3tsconfig:mod-wizards-newcontentelement-wizarditems-group-elements-name-saveandclose>`
 
-   mod.wizards.newContentElement.wizardItems.common {
-       elements {
-           example_registration {
-               iconIdentifier = example-registration
-               title = Example title
-               description = Example description
-               tt_content_defValues {
-                   CType = list
-                   list_type = example_registration
-               }
-           }
-       }
-       show := addToList(example_registration)
-   }
+..  literalinclude:: _AddingYourOwnContentElements/_page_change_item.tsconfig
+    :caption: EXT:my_sitepackage/Configuration/Sets/MySet/page.tsconfig
 
-When you look at existing page TSconfig in the :guilabel:`Info` module, you may
-notice that :typoscript:`show` has been set to include all for the
-:guilabel:`Plugins` tab:
+.. _content-element-wizard-create-group:
 
-.. code-block:: typoscript
+Register a new group in the "New Content Element" wizard
+========================================================
```

Comment:
Note: The tabs/groups are ordered in ASC order by their group keys (not label) by default. With "before" and "after" you can move a group to desired position.

Reply:
Actually this does not work with pure TCA, see https://typo3.slack.com/archives/C03AM9R17/p1725799806264659

Reply:
I would adress this topic in a follow up depending on how the fixes develop